curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3952facd25a83fe8083.md

```diff
@@ -7,35 +7,31 @@ dashedName: step-20
 
 # --description--
 
-Create a media query for screens smaller than `800px` in width. In that media query, create a `#gallery img` rule and set the `width` property to `50%`. This will convert your gallery to a two-column layout.
+Because we set `justify-content: center` on the flex container the images will be centered. The `::after` pseudo-element creates an element that is the last child of the selected element. We can use it to add an empty element after the last image. If we give it the same `width` as the images it will push the last image to the left when the gallery is in a two-column layout.
 
-# --hints--
-
-You should add a new `@media` query.
+Example:
 
-```js
-assert(new __helpers.CSSHelp(document).getCSSRules('media')?.length === 1);
+```CSS
+.container::after {
+  content: "";
+  width: 860px;
+}
 ```
 
-Your new `@media` query should have a `max-width` of `800px` like this: `@media (max-width: 800px)`.
+Create a new selector using an `::after` pseudo-element on the `.gallery` element. Add a `content` property set to an empty string `""` and `350px` set for the `width` property.
```

Comment:
```suggestion
Create a new selector using an `::after` pseudo-element on the `.gallery` element. Add an empty `content` property and `width` of `350px` as its value.
```

Reply:
The `content` property isn't empty, it is an empty string. An empty property would suggest the value is missing.